PanoptesNetClient/PanoptesNetClient/Auth.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Net.Http;
+
+namespace PanoptesNetClient
+{
+    static class Constants
+    {
+        public const int BearerTokenExpirationAllowance = 60 * 1000;
+    }
+
+    public class Auth
+    {
+        private string BearerToken;
+        private double BearerTokenExpiration;
+        private string RefreshToken;
+        private dynamic TokenRefreshPromise;
+        private static readonly HttpClient client = new HttpClient();
+
+        private void _GetAuthToken()
```

Comment:
Is the underscore a normal C# naming convention? We use it in js to denote private functions, but I don't think we need it in c# because you use the `private` keyword to denote private functions.